src/client/Ruby/ms-rest-azure/lib/ms_rest_azure/polling_state.rb

```diff
@@ -109,17 +116,5 @@ def get_request(options = {})
 
     attr_accessor :connection
 
-    def create_connection(base_url)
-      @connection ||= Faraday.new(:url => base_url, :ssl => MsRest.ssl_options) do |faraday|
-        [[MsRest::RetryPolicyMiddleware, times: 3, retry: 0.02], [:cookie_jar]].each{ |args| faraday.use(*args) }
-        faraday.adapter Faraday.default_adapter
-        faraday.headers = request.headers
-        logging = ENV['AZURE_HTTP_LOGGING'] || request.log
-        if logging
-          faraday.response :logger, nil, { :bodies => logging == 'full' }
-        end
-      end
-    end
   end
-
 end
```

Comment:
Again this is not regarding your change but i just notices. We should add "a new line character" on line 120[last line].